website/docs/r/iotcentral_application.html.markdown

```diff
@@ -0,0 +1,69 @@
+---
+subcategory: "IoT Central"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_iotcentral_application"
+description: |-
+  Manages an IotCentral Application
+---
+
+# azurerm_iotcentral_application
+
+Manages an IotCentral Application
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "resourceGroup1"
+  location = "West US"
```

Comment:
location = "West Europe" ,right?

Reply:
code rule , location should only be 'West Europe' @xuzhang3